lib/dry/validation/messages/i18n.rb

```diff
@@ -18,7 +18,8 @@ def get(key, options = {})
       end
 
       def key?(key, options)
-        ::I18n.exists?(key, options.fetch(:locale, default_locale))
+        ::I18n.exists?(key, options.fetch(:locale, default_locale)) ||
+        ::I18n.exists?(key, I18n.default_locale)
```

Comment:
Hi! the reason behind this is because could be the case that someone has `I18n` configured to use [fallbacks](https://github.com/svenfuchs/i18n/wiki/Fallbacks) but the `exists?`call return `false` on the `key?` and so never fetch afterwards. 

With this extra check, the `get(key, options = {})` will call `translate` on the I18n module and it will handle the fallback properly.

Also I know that is not as accurate as it could be, because we are not necessary checking on all the fallback chain, but if someone has I18n fallback configured with a default one probably this will have at least all the keys, I'm assuming this though 😕 

I just push a spec that shows the reason behind this